cartesian_trajectory_interpolation/include/cartesian_trajectory_interpolation/cartesian_trajectory_segment.h

```diff
@@ -142,7 +142,8 @@ std::ostream& operator<<(std::ostream& os, const CartesianTrajectorySegment::Spl
  *
  * The computation of quaternion velocities and accelerations from
  * Cartesian angular velocities and accelerations is based on
- * <a href="https://math.stackexchange.com/questions/1792826">this blog post</a>.
+ * <a href="https://math.stackexchange.com/questions/1792826">this blog post</a> and on <a
+ * href="https://arxiv.org/abs/0811.2889v1">this article</a>. 
```

Comment:
@urmahp Could be a trailing white space that causes the linting to fail..

Reply:
Yes that was the issue, thank you.